src/commands/misc/npm.js

```diff
@@ -0,0 +1,31 @@
+const { Command, SwitchbladeEmbed, Constants } = require('../../')
+const npm = require('api-npm')
+
+module.exports = class Npm extends Command {
+  constructor (client) {
+    super(client)
+    this.name = 'npm'
+  }
+
+  run (message, args) {
+    const embed = new SwitchbladeEmbed(message.author)
+    if (args[0]) {
+      npm.getdetails(args.join('-'), data => {
+        if (data.name) {
+          embed.setAuthor(data.name, 'https://i.imgur.com/24yrZxG.png', 'https://www.npmjs.com/package/' + data.name)

```

Comment:
In + data.name can be used `https://www.npmjs.com/package/${data.name}`

Reply:
ik